bazel/repositories_extra.bzl

```diff
@@ -15,7 +15,7 @@ PYTHON_MINOR_VERSION = _python_minor_version(PYTHON_VERSION)
 # Envoy deps that rely on a first stage of dependency loading in envoy_dependencies().
 def envoy_dependencies_extra(
         python_version = PYTHON_VERSION,
-        ignore_root_user_error = False):
+        ignore_root_user_error = True):
```

Comment:
cc @phlax
I think we discussed this a while ago (maybe I'm misremembering).

@daixiang0 why is this needed?
Can this be added to a specific dependency, and not impact all dependencies?

Reply:
yeah - we dont want this

Reply:
i dont think this is related to the PR tbh - its related to local setup